final release/format_data.py

```diff
@@ -1,200 +1,266 @@
-import numpy as np
+"""Functions for formatting, preprocessing, and augmentation."""
+
+import sys, os
```

Comment:
Split this into two imports (separate lines) (GS)

Reply:
ok